plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/helpers.ts

```diff
@@ -351,3 +355,22 @@ function extractCollaboratorName(
   if ('user' in collaborator) return collaborator.user;
   return collaborator.team;
 }
+
+async function validateAccessTeam(client: Octokit, access: string) {
+  const [org, team_slug] = access.split('/');
+  try {
+    // Below rule disabled because of a 'getByName' check for a different library
+    // incorrectly triggers here.
+    // eslint-disable-next-line testing-library/no-await-sync-query
+    await client.rest.teams.getByName({
+      org,
+      team_slug,
+    });
+  } catch (e) {
+    if (e.response.data.message === 'Not Found') {
+      const message = `Received 'Not Found' from the API; one of org:
+        ${org} or team: ${team_slug} was not found within GitHub.`;
+      throw new Error(message, { cause: e });
```

Comment:
`ForwardedError` from `@backstage/errors` provides a pattern for this as well. In this case I actually think we may want to ignore the cause and create a new error though. In that case probably a `NotFoundError`.

Reply:
Addressed in https://github.com/backstage/backstage/pull/16059/commits/23f21636358524615a23ed962cff33d296f7d82d

Reply:
... and then removed cause in https://github.com/backstage/backstage/pull/16059/commits/67ed63552446b512d549e911e13cd7d36eb36186 😅